src/commands/secrets/download.ts

```diff
@@ -38,7 +41,17 @@ export default class SecretsDownload extends BaseCommand {
       this.error('You do not have permission to download secrets. Please contact your admin.');
     }
 
-    const secrets = await SecretUtils.getSecrets(this.app, account, flags.environment, true);
+    if (flags.platform && flags.environment) {
+      throw new Error('Please provide either the platform flag or the environment flag and not both.');
+    }
+
+    let secrets;
```

Comment:
This needs to be const. If we are going to tell the user that something does not exist, then we should check it and make sure instead of waiting on an error to be thrown.